webcompat/form.py

```diff
@@ -34,11 +34,15 @@
 
 problem_choices = [
     (u'detection_bug', u'Desktop site instead of mobile site'),
-    (u'mobile_site_bug', u'Mobile site is not usable'),
-    (u'video_bug', u'Video doesn\'t play'),
-    (u'layout_bug', u'Layout is messed up'),
-    (u'text_bug', u'Text is not visible'),
-    (u'unknown_bug', u'Something else - I\'ll add details below')
+    (u'mobile_site_bug', u'Site is not usable'),
```

Comment:
I don't want to bikeshed `problem_choices` naming, but "Site is not usable" will get used for desktop sites as well. I'm not sure it matters much though?